README.md

```diff
@@ -1,43 +1,58 @@
-# eslint-import-resolver-kibana
+# @elastic/eslint-import-resolver-kibana
 
 Resolver for Kibana imports, meant to be used with [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import).
 
 ## Usage
 
-In your `.eslintrc.(yml|json|js)` file, add the following `import/resolver` under `settings`:
+Specify this resolver at `settings["import/resolver']` in your eslint config file:
```

Comment:
not necessarily, I'm really just trying to describe the property. The copiable code snippet is below, but I can use a more prose version.